src/io.rs

```diff
@@ -483,7 +403,7 @@ impl RustyBuffer {
 
         (*view).buf = bytes.as_ptr() as *mut std::os::raw::c_void;
         (*view).len = bytes.len() as isize;
-        (*view).readonly = 1;
+        (*view).readonly = 0;
```

Comment:
Interesting. Is this flag visible from Python? If it is, it was forcing a deep-copy upon deserialization; see:
https://github.com/dask/distributed/blob/4bd2ba7ec4e4a4669c13dc1e62820191664783f4/distributed/protocol/numpy.py#L137-L148

Reply:
Depends on the consumer of the buffer protocol. The consumer can choose to respect this (most likely) otherwise there is nothing truly stopping it from modifying the buffer. So likely, as seen there, if it needs to modify it, then it'll make a copy. I saw numpy defaults to allow it, so followed suit here and needed anyway as `cramjam.Buffer` can be passed in as before but now using the buffer protocol rather than the object itself.